scripts/integration/mqtt/compose.yaml

```
@@ -0,0 +1,11 @@
+version: '3'
+
+services:
+  emqx:
+    image: docker.io/emqx:${MQTT_VERSION}
+    ports:
+    - 1883:1883
```

```suggestion
    - 18083:18083
```

I suspect that is the reason it isn't starting

---

Fixed.